amf-cli/shared/src/test/resources/validations/async20/validations/mercure-binding-nonempty.yaml

```diff
@@ -7,25 +7,21 @@ servers:
     url: some.com
     protocol: mercure
     bindings:
-      mercure: {
+      mercure:
```

Comment:
This commits should be squashed into the one that introduced this file as it is a fix over the same file with the same content.

Reply:
done, reorganized commits